### PR TITLE
Formfield

### PR DESCRIPTION
**Pull request by lancondrej**
I add a simple formfield (text input), otherwise, it shows a default str for relativedelta, but this is not a valid input for RelativeDeltaField, so I use ISO8601 formatting also for showing value.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-sudo: false
+s	sesudo: false
```
Comment (sjamaan) [inline .travis.yml]: Could you explain what this does? Looks like an accidental commit to me.

File: relativedeltafield/fields.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	def get_db_converters(self, connection):
+		return [self.convert_relativedeltafield_value]
+
+	def convert_relativedeltafield_value(self, value, expression, connection, context):
+		if value is not None:
+			return parse_relativedelta(value)
+
+	def value_to_string(self, obj):
+		val = self.value_from_object(obj)
+		return '' if val is None else format_relativedelta(val)
```
Comment (sjamaan) [inline relativedeltafield/fields.py]: It looks like you reindented the entire file, causing it to be seen as completely changed. That makes it nigh-impossible for me to review. I know tabs for indentation is nonstandard, but please keep the indentation style the way it is.

Note that there is a `.editorconfig` in the root, which should be picked up by your editor if it's configured to understand [editorconfig](https://editorconfig.org/). That should fix a lot of these issues!

File: relativedeltafield/tests/test_relativedeltafield.py [hunk 3]
```diff
@@ -186,7 +176,6 @@ def test_timedelta_input(self):
 		# Derived value, from the number of days (see #2)
 		self.assertStrictEqual(1, obj.value.weeks)
 
-
 	def test_filtering_works(self):
 		obj1 = Interval(value='P1Y3M1W4.5DT5H70.5M80.10001S')
 		obj1.save()
```
Comment (sjamaan) [inline relativedeltafield/tests/test_relativedeltafield.py]: Also in this file there are a lot of unnecessary whitespace changes.

I'd appreciate it if you could also add actual tests for the form field.

**Comment by sjamaan** [comment]
Argh, Travis is still failing because Django 2 doesn't work with old Pythons

File: .travis.yml (hunk 1 above)
Comment (lancondrej) [reply .travis.yml]: Yes it was a mistake, already fixed

File: relativedeltafield/fields.py (hunk 2 above)
Comment (lancondrej) [reply relativedeltafield/fields.py]: Sorry, I did not notice the editorconfig, I will reformat it back. The problem is that I also change a structure a little bit (I don't like so much code in init file :wink: )

**Comment by sjamaan** [comment]
Could you please rebase your changes against `master`? Also, I think you haven't fixed the indentation changes yet.

**Comment by sjamaan** [comment]
Months have passed without any response or update. If you'd like to have this merged, please rebase and minimise the changes so it's easier to review. You can make a new PR or re-open this one then.
